app/models/user.rb

```diff
@@ -12,4 +12,8 @@ class User < ApplicationRecord
   validates :username, presence: true, uniqueness: { case_sensitive: false }
   validates :email, presence: true, uniqueness: { case_sensitive: false }
   validates :password, presence: true
+
+  def self.search(search_params)
```

Comment:
May need to add a class method test for this in the user model spec. Honestly not sure though since we test this search method so thoroughly in the other spec test you built.